server/schedule/operator/operator.go

```diff
@@ -98,6 +98,16 @@ func (s StoreInfluence) ResourceSize(kind core.ResourceKind) int64 {
 	}
 }
 
+type u64Set map[uint64]struct{}
+
+func (s u64Set) String() string {
+	v := make([]uint64, 0, len(s))
+	for x := range s {
+		v = append(v, x)
+	}
+	return fmt.Sprintf("%v", v)
```

Comment:
How about output in a comma-separated list form? How about sort the slice before output?

Reply:
I will sort the slice. However, I think the comma is not necessary, because all elements are numbers, using space to separate them is clear enough.

Reply:
ok, got it.